ftw/usermigration/browser/migration.pt

```diff
@@ -16,6 +16,44 @@
 <metal:content-core fill-slot="content-core">
     <metal:content-core define-macro="content-core">
 
+
+    <tal:pre-migration-hooks define="results_pre_migration view/results_pre_migration"
+                             repeat="hook_name python:results_pre_migration.keys()"
+                             condition="view/results_pre_migration">
+
+    <h2>Pre-Migration Hooks</h2>
+
+    <h3 tal:content="hook_name" />
+
+    <tal:hook_results define="hook_results python:results_pre_migration[hook_name]"
```

Comment:
@jone yes and no :) I wanted to keep the style in line with the rest of the template and *not* indent pure `tal:` grouping blocks that aren't visible in the rendered output, but I indented one level by accident. So I dedented that one. But this template is in need of a major refactoring anyway, I'll address that in one of the next PRs.